src/components/timeline/TimelineContextMenu.svelte

```diff
@@ -285,25 +298,44 @@
     </ContextMenuItem>
   {/if}
   <ContextMenuSeparator />
-  <ContextSubMenuItem
-    text={`Zoom${activityDirective ? ' around Selected Directive' : ''}`}
-    parentMenu={contextMenuComponent}
-  >
-    <ContextMenuItem on:click={() => onZoomHome()}>Reset Zoom</ContextMenuItem>
-    <ContextMenuSeparator />
-    <ContextMenuItem on:click={() => onZoom(TIME_MS.MILLISECOND)}>Millisecond</ContextMenuItem>
-    <ContextMenuItem on:click={() => onZoom(TIME_MS.MILLISECOND * 10)}>10 Milliseconds</ContextMenuItem>
-    <ContextMenuItem on:click={() => onZoom(TIME_MS.MILLISECOND * 50)}>50 Milliseconds</ContextMenuItem>
-    <ContextMenuItem on:click={() => onZoom(TIME_MS.SECOND)}>Second</ContextMenuItem>
-    <ContextMenuItem on:click={() => onZoom(TIME_MS.SECOND * 30)}>30 Seconds</ContextMenuItem>
-    <ContextMenuItem on:click={() => onZoom(TIME_MS.MINUTE)}>Minute</ContextMenuItem>
-    <ContextMenuItem on:click={() => onZoom(TIME_MS.MINUTE * 30)}>30 Minutes</ContextMenuItem>
-    <ContextMenuItem on:click={() => onZoom(TIME_MS.HOUR)}>Hour</ContextMenuItem>
-    <ContextMenuItem on:click={() => onZoom(TIME_MS.HOUR * 12)}>12 Hours</ContextMenuItem>
-    <ContextMenuItem on:click={() => onZoom(TIME_MS.DAY)}>Day</ContextMenuItem>
-    <ContextMenuItem on:click={() => onZoom(TIME_MS.DAY * 3)}>3 Days</ContextMenuItem>
-    <ContextMenuItem on:click={() => onZoom(TIME_MS.DAY * 7)}>Week</ContextMenuItem>
-    <ContextMenuItem on:click={() => onZoom(TIME_MS.MONTH)}>Month</ContextMenuItem>
-    <ContextMenuItem on:click={() => onZoom(TIME_MS.YEAR)}>Year</ContextMenuItem>
-  </ContextSubMenuItem>
+  {#if span}
+    <ContextSubMenuItem text="Center around Simulated Activity" parentMenu={contextMenuComponent}>
```

Comment:
Thoughts on "Center around" vs "Zoom to" for span vs directive? Know they're a little different behavior wise but wondering if we could get away with the same phrasing for both?

Reply:
Yeah that's a tough one. I asked @lklyne for some feedback.

Reply:
renamed to "zoom". thanks @lklyne!